src/main/java/com/eppo/sdk/helpers/bandit/BanditEvaluator.java

```diff
@@ -17,14 +14,15 @@ public class BanditEvaluator {
 
     public static List<Variation> evaluateBanditActions(
         String experimentKey,
-        String modelName,
+        BanditParameters modelParameters,
         Map<String, EppoAttributes> actions,
         String subjectKey,
         EppoAttributes subjectAttributes,
         int subjectShards
     ) {
+        String modelName = modelParameters != null ? modelParameters.getModelName() : "random";
         BanditModel model = BanditModelFactory.build(modelName);
-        Map<String, Float> actionWeights = model.weighActions(actions, subjectAttributes);
```

Comment:
Changed  internal weight calculations from `Float` to `Double` to match the type we use for numeric `EppoValue`s